bundles/org.openhab.binding.insteon/src/main/resources/device_types.xml

```diff
@@ -76,6 +76,8 @@ Example entry:
      <model>2842-222</model>
      <description>Motion Sensor</description>
      <feature name="contact">WirelessMotionSensorContact</feature>
+     <feature name="lightlevelabovethreshold">WirelessMotionSensorLightLevelAboveThreshold</feature>
+     <feature name="lowbattery">WirelessMotionSensorLowbattery</feature>
```

Comment:
```suggestion
     <feature name="lowbattery">WirelessMotionSensorLowBattery</feature>
```

Reply:
No problem.  Sorry for the issues.  Done now, pushed new version.